framework/include/math_utils.hpp

```diff
@@ -12,6 +12,10 @@ namespace gvk
 	/// extract the translation part out of a matrix
 	extern glm::vec3 get_translation_from_matrix(const glm::mat4& m);
 
+	glm::mat4 cancel_translation_from_matrix(const glm::mat4& m);
```

Comment:
Please add comments in the following style:
 ```
/** Function description
 *   @param  aMatrix   Parameter description
 *   @return Return value description
 */
glm::mat4 cancel_translation_from_matrix(const glm::mat4& aMatrix);
```
I know that most of the functions in `math_utils.hpp` are not well-behaved in that regard, so please don't let yourself be guided by them but follow the coding guidelines!